README.md

```diff
@@ -1,17 +1,59 @@
-# tailscale-client-go
+// Copyright (c) David Bond, Tailscale Inc, & Contributors
```

Comment:
In other repos, e.g., OSS, we've done this by including a reference to the AUTHORS file and listing contributors there. That might be worth preserving here.

Reply:
We had some internal discussion with @willnorris about this back in September and settled on what we have for the OG tailscale-client-go repo. I'm inclined to leave that as-is here.